applications/sink/zeromq-sink/src/main/resources/dataflow-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1 @@
+configuration-properties.classes=org.springframework.cloud.fn.consumer.zeromq.ZeroMqConsumerProperties
```

Comment:
Both of these files have to be in the `/META-INF` directory.

See all other apps.
The README should be generated properly.